easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb

```diff
@@ -0,0 +1,48 @@
+# This file is an EasyBuild reciPY as per http://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'ConfigureMake'
+
+name = 'LUMPY'
+version = '0.2.13'
+
+homepage = 'https://github.com/arq5x/lumpy-sv'
+description = 'lumpy: a general probabilistic framework for structural variant discovery'
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+# no source_urls because the tarball has to be manually generated doing 
+# git clone --recursive https://github.com/arq5x/lumpy-sv.git
+sources = ['%(name)s-%(version)s.tar.gz']
+
+unpack_options = '--strip-components=1'
+
+builddependencies = [
+    ('CMake', '3.4.3'),
+    ('Autotools', '20150215', '', True),
+]
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('zlib', '1.2.8'),
+    ('samblaster', '0.1.24'),
+    ('SAMtools', '1.3.1'),
+    ('Sambamba', '0.6.6', '', True),
+]
+
+osdependencies = [('gawk')]
```

Comment:
Why not as a proper dep through EasyBuild? We have (old) easyconfigs for `gawk`...